sdk/python/examples/quickstart/Quickstart.ipynb

```diff
@@ -0,0 +1,239 @@
+   },
+   "outputs": [],
+   "source": [
+    "# Now that we have finished creating our features, we ingest them into feast\n",
+    "\n",
+    "# Initialise client\n",
+    "fs = Client(core_url=FEAST_CORE_URL, verbose=True)\n",
+    "\n",
+    "# Create importer\n",
+    "importer = Importer.from_df(df_complete, \n",
+    "                           entity='ride', \n",
+    "                           granularity=Granularity.NONE,\n",
+    "                           owner='user@website.com',  \n",
+    "                           staging_location=STAGING_LOCATION,\n",
+    "                           id_column='ride', \n",
+    "                           timestamp_column='pickup_datetime')\n",
+    "\n",
+    "# Update feature and entity metadata\n",
+    "serving_ds=Datastore(id='REDIS1').spec\n",
+    "warehouse_ds=Datastore(id='NOOP').spec\n",
+    "\n",
+    "# Update feature and entity metadata. Ideally you want to update these manually\n",
+    "# so that they contain adequate information for the next user\n",
+    "importer.entity.description = 'nyc taxi dataset' \n",
+    "for feature in importer.features:\n",
+    "    feature.serving_store = serving_ds\n",
+    "    feature.warehouse_store = warehouse_ds\n",
+    "    feature.description = 'nyc taxi dataset'\n",
+    "    \n",
+    "# Ingest the feature data into the store\n",
+    "fs.run(importer, apply_features=True, apply_entity=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating a training dataset\n",
+    "\n",
+    "Creating a training dataset allows you to isolate the data that goes into the model training step, allowing for reproduction and traceability."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Retrieving data: Training\n",
+    "\n",
+    "feature_set = FeatureSet(entity=\"ride\", \n",
+    "                         features=[\"ride.none.log_trip_duration\", \n",
+    "                                  \"ride.none.distance_haversine\",\n",
+    "                                  \"ride.none.distance_dummy_manhattan\",\n",
+    "                                  \"ride.none.direction\",\n",
+    "                                  \"ride.none.month\",\n",
+    "                                  \"ride.none.day_of_month\",\n",
+    "                                  \"ride.none.hour\",\n",
+    "                                  \"ride.none.day_of_week\",\n",
+    "                                  \"ride.none.vi_1\",\n",
+    "                                  \"ride.none.vi_2\",\n",
+    "                                  \"ride.none.sf_n\",\n",
+    "                                  \"ride.none.sf_y\"])\n",
+    "dataset_info = fs.create_training_dataset(feature_set, \"2016-06-01\", \"2016-08-01\")\n",
+    "dataset = fs.download_dataset_to_df(dataset_info, staging_location=STAGING_LOCATION)\n",
```

Comment:
Same as above. `staging_location` seems unnecessary.

Also, we should probably have a `wrapper` command which  does both `create_training_dataset` and `download_dataset_to_df` in one go. 

```
dataset = fs.create_training_dataset_to_df(feature_set, "2016-06-01", "2016-08-01")
```

which is a little bit nicer because the user doesn't have to deal with `dataset_info` as an intermediate step as in
```
dataset_info = fs.create_training_dataset(feature_set, \"2016-06-01\", \"2016-08-01\")
dataset = fs.download_dataset_to_df(dataset_info, staging_location=STAGING_LOCATION)
```

Also, now that I think about it. What do you think about removing the word `training` completely from this API? So we just have `create_dataset`?

Reply:
Imo I think its fine to keep the two methods separate. 
Good point, will rename the method to `create_dataset`.